types/react-virtual-keyboard/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for react-virtual-keyboard 1.0
+// Project: https://www.npmjs.com/package/react-virtual-keyboard
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
```

Comment:
@sheetalkamat I'll write to react-virtual-keyboard authors. They have to change it.